RetailCoder.VBE/Inspections/InspectionsUI.resx

```diff
@@ -304,7 +304,7 @@
     <value>Variable is used but not assigned</value>
   </data>
   <data name="UntypedFunctionUsageInspectionMeta" xml:space="preserve">
-    <value>A string-returning equivalent function exists and should preferably be used to avoid implicit type conversions.</value>
+    <value>A string-returning equivalent function exists and should preferably be used to avoid implicit type conversions. However, this function will crash if passed a 'Null' value, so do not use it if the value passed can be 'Null'.</value>
```

Comment:
May I suggest a more.. tactful wording?

> If the parameter can be null, ignore this inspection result; passing a null value to a function expecting a string would raise a type mismatch runtime error.

The UI should be able to handle the wordiness.